prusti-interface/src/utils.rs

```diff
@@ -263,6 +263,34 @@ impl<'tcx> VecPlace<'tcx> {
     }
 }
 
+/// Check if `prusti::<name>` is among the attributes.
+/// The attribute must not have any arguments.
```

Comment:
What happens if the attribute has arguments? The function returns `false`?

Reply:
Yes.